### ISSUE TITLE
Training loop stuck in dataloader

### ISSUE THREAD

**Issue by piseabhijeet** [LongLong-Jing/Cross-Modal-Center-Loss#3]
Hi @LongLong-Jing 

Thank you for the amazing repo!

I was trying to run the training code and it seems that it is stuck at this point:

![image](https://user-images.githubusercontent.com/57705684/124543054-2aab2880-de42-11eb-9c9c-4b2bf6222c4d.png)

Nothing happens for long time and only 1KB summaries are created.

**Comment by piseabhijeet** [comment]
If data is not available for any of the modalities, the dataloader will get stuck in the while loop.
